export/policy.ts

```diff
@@ -7,7 +7,7 @@ async function fetchPolicyRequest(url: string): Promise<string> {
     const response = await PhylumApi.fetch("v0/", url, {});
     
     if(!response.ok) {
-        console.error(`\nFailed to get the policy for project '${projectId}'`);
+        console.error(`\nFailed to get the policy for project`);
```

Comment:
```suggestion
        console.error(`\nFailed to get the policy`);
```
Specifying "for project" is too specific since it may have also failed to get the policy for the group.